ansible/roles/cleaner/tasks/main.yml

```diff
@@ -45,6 +45,10 @@
     force: yes
   ignore_errors: yes
 
+- name: clean device
```

Comment:
I think you should add some pre-conditions here because not all backend need local `loop*` device.